sdk/keyvault/keyvault-keys/test/public/list.spec.ts

```diff
@@ -9,239 +9,255 @@ import { assertThrowsAbortError } from "../utils/utils.common";
 import { testPollerProperties } from "../utils/recorderUtils";
 import { authenticate } from "../utils/testAuthentication";
 import TestClient from "../utils/testClient";
+import { supports, versionsToTest, SupportedVersions } from "@azure/test-utils-multi-version";
 
-describe("Keys client - list keys in various ways", () => {
-  const keyPrefix = `list${env.KEY_NAME || "KeyName"}`;
-  let keySuffix: string;
-  let client: KeyClient;
-  let testClient: TestClient;
-  let recorder: Recorder;
-
-  beforeEach(async function() {
-    const authentication = await authenticate(this);
-    keySuffix = authentication.keySuffix;
-    client = authentication.client;
-    testClient = authentication.testClient;
-    recorder = authentication.recorder;
-  });
-
-  afterEach(async function() {
-    await recorder.stop();
-  });
-
-  // The tests follow
-
-  // Use this while recording to make sure the target keyvault is clean.
-  // The next tests will produce a more consistent output.
-  // This test is only useful while developing locally.
-  it("can purge all keys", async function(): Promise<void> {
-    // WARNING: When TEST_MODE equals "record", all of the keys in the indicated KEYVAULT_NAME will be deleted as part of this test.
-    if (!isRecordMode()) {
-      return this.skip();
-    }
-    for await (const properties of client.listPropertiesOfKeys()) {
-      try {
-        await testClient.flushKey(properties.name);
-      } catch {
-        // Nothing to do here.
-      }
-    }
-    for await (const deletedKey of client.listDeletedKeys()) {
-      try {
-        await testClient.purgeKey(deletedKey.name);
-      } catch {
-        // Nothing to do here.
-      }
-    }
-  });
-
-  it("can get the versions of a key", async function() {
-    const keyName = testClient.formatName(`${keyPrefix}-${this!.test!.title}-${keySuffix}`);
-    await client.createKey(keyName, "RSA");
-    let totalVersions = 0;
-    for await (const version of client.listPropertiesOfKeyVersions(keyName)) {
-      assert.equal(
-        version.name,
-        keyName,
-        "Unexpected key name in result from listPropertiesOfKeyVersions()."
+versionsToTest(["7.0", "7.1"]).forEach((serviceVersion) => {
+  const versions = function(versions: SupportedVersions) {
+    return supports(serviceVersion, versions);
+  };
+
+  versions(["7.0", "7.1"]).describe.only(
```

Comment:
nit: `only` here is a testing artifact, right?

Reply:
This is just to test that `.describe.only()` still work as expected.  For keyvault-keys, as far as I know we don't actually need `versions()` at all because all the tests will run for the two versions without any behavior difference.

Reply:
Looks like I may also need `.skip` on the chain object.